packages/rpc/src/client.ts

```diff
@@ -469,49 +483,42 @@ export class RPCClient extends EventEmitter {
 
 	/**
 	 * Get current voice settings
-	 * @returns {Promise<GetVoiceSettingsPayloadData>}
 	 */
-	public getVoiceSettings(): Promise<GetVoiceSettingsPayloadData> {
-		return this.request(RPCCommands.GetVoiceSettings).then((s) => s as GetVoiceSettingsPayloadData);
+	public async getVoiceSettings(): Promise<GetVoiceSettingsPayloadData> {
+		return this.request(RPCCommands.GetVoiceSettings) as Promise<GetVoiceSettingsPayloadData>;
 	}
 
 	/**
 	 * Set current voice settings, overriding the current settings until this session disconnects.
 	 * This also locks the settings for any other rpc sessions which may be connected.
-	 * @param {GetVoiceSettingsPayloadData} args Settings
-	 * @returns {Promise}
+	 *
+	 * @param args - Settings
 	 */
-	public setVoiceSettings(args: GetVoiceSettingsPayloadData): Promise<unknown> {
+	public async setVoiceSettings(args: GetVoiceSettingsPayloadData): Promise<unknown> {
 		return this.request(RPCCommands.SetVoiceSettings, args);
 	}
 
 	/**
-	 * Capture a shortcut using the client
-	 * The callback takes (key, stop) where `stop` is a function that will stop capturing.
-	 * This `stop` function must be called before disconnecting or else the user will have
-	 * to restart their client.
-	 * @param {Function} callback Callback handling keys
-	 * @returns {Promise<Function>}
+	 * Start capturing a shortcut using the client
 	 */
-	public captureShortcut(callback: (key: string, stop: () => Promise<unknown>) => void) {
-		const subid = subKey(RPCEvents.CaptureShortcutChange);
-		const stop = () => {
-			this._subscriptions.delete(subid);
-			return this.request(RPCCommands.CaptureShortcut, { action: 'STOP' });
-		};
-		this._subscriptions.set(subid, ({ shortcut }) => {
-			callback(shortcut, stop);
-		});
-		return this.request(RPCCommands.CaptureShortcut, { action: 'START' }).then(() => stop);
+	public async startCapturingShortcut() {
+		return this.request(RPCCommands.CaptureShortcut, { action: 'START' });
```

Comment:
Another enum for the `action`? 😄

Reply:
I don't think you need it since it's used intenrally only